docs/concepts/architecture/apiserver-aggregation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Extending the Kubernetes API with Aggregator
+assignees:
+- chenopis
+---
+
+{% capture overview %}
+
+The *Aggregator* is a system to allow the Kubernetes apiserver to be extended with additional APIs, which are not part of the core Kubernetes APIs. 
```

Comment:
Is it best practice to refer to the "apiserver" or "API Server"? I've seen it both ways---it's more of a nit, but consistency would be nice.